setup.py

```diff
@@ -14,7 +14,8 @@ def readme():
     'MatPlot': ('matplotlib', '2.2.3'),
     'QtPlot': ('pyqtgraph', '0.10.0'),
     'coverage tests': ('coverage', '4.0'),
-    'Slack': ('slacker', '0.9.42')
+    'Slack': ('slacker', '0.9.42'),
+    'ZurichInstruments': ('zhinst-qcodes', '0.1.1')
```

Comment:
I would prefer not to depend on the ZI modules. There are many users that do not currently use ZI drivers. Can we perhaps replace the raise of an Import error with a warning.

Reply:
This is an optional dependency, it's installed only if the user ask explicitly for it with something like 'pip install qcodes[ZurichInstrument]'. There is anyway an import check in the individual drivers.

Reply:
Sorry I was too quick. This does indeed look fine to me